src/components/Navigation/ConversationsList/AppContentListItem/AppContentListItem.vue

```diff
@@ -148,6 +148,11 @@ export default {
 	justify-content: space-between;
 	align-items: center;
 	flex: 0 0 auto;
+	cursor: pointer;
+	// Make the cursor a ponter on every child element
+	* {
```

Comment:
Hum, no.
This is usually when shit happens :p
Please properly target the needed entries